Dockerfile

```diff
@@ -216,6 +218,14 @@ RUN curl -fsSL https://ftp.ncbi.nlm.nih.gov/pub/datasets/command-line/v2/linux-$
 RUN curl -fsSL https://ftp.ncbi.nlm.nih.gov/pub/datasets/command-line/v2/linux-${TARGETARCH}/dataformat \
   -o /final/bin/dataformat
 
+# Install tooling for our AWS Batch builds, which use `aws s3`.
+# Using `xx-info march` to translate amd64 -> x86_64 and arm64 -> aarch64
+# based on https://github.com/docker/buildx/discussions/2001
+RUN XX_MARCH=$(xx-info march) \
+  && curl "https://awscli.amazonaws.com/awscli-exe-linux-${XX_MARCH}.zip" -o "awscliv2.zip" \
+  && unzip awscliv2.zip \
+  && ./aws/install --install-dir /final/bin/aws-cli --bin-dir /final/bin
```

Comment:
On the face of it, I'd suggest this change

```suggestion
  && ./aws/install --install-dir /final/libexec/aws-cli --bin-dir /final/bin
```

which will place the internal executable libraries in a more conventional place, leaving the `bin` dir for just the user-invoked programs.

However, an additional change is needed to fix the broken symlink issue you faced in https://github.com/nextstrain/docker-base/pull/214/commits/f06e64cb54b8e93d42c4486460a5137f6195d0e3, because the installer (`./aws/install`) creates the symlinks with absolute paths like so:

    ln -s /final/libexec/aws-cli/v2/x.y.z /final/libexec/aws-cli/v2/current
    ln -s /final/libexec/aws-cli/v2/current/bin/aws /final/bin/aws

and those target paths (the first paths) don't exist after copying the symlinks (the second paths) into the final image.

We could play tricks on the installer to get it to do what we want by arranging the cwd and `--install-dir` with a relative path just so, but that feels brittle and unobvious. Instead, I'd suggest just installing the files ourselves:

```suggestion
  && mv ./aws/dist /final/libexec/aws-cli \
  && ln -srv /final/libexec/aws-cli/aws /final/bin/aws
```

Also possible to `unzip` directly into place instead of doing `unzip` + `mv`.

Reply:
> Instead, I'd suggest just installing the files ourselves

Ah, would we not be worried about the filepaths changing? I guess it's pretty unlikely they would change `./aws/dist`.

Reply:
Seems unlikely to change to me, and also seems less likely to break compared to a carefully constructed just-so invocation of `./aws/install` that tightly depends on the internals of `./aws/install`.

Reply:
Updated in https://github.com/nextstrain/docker-base/pull/214/commits/ef3577f9e1452325ad4280b64c94082f37079e06